api/reference/network_routers.adoc

```diff
@@ -0,0 +1,71 @@
+
+[[network-routers]]
+== Network Routers
+
+Management of Network Routers is provided via the following collection:
+
+[source,data]
+----
+/api/network_routers
+----
+
+The following actions are available on network_routers:
```

Comment:
I noticed that some collection documentation refers to the actual collection name (href reference) while some reference them as their real name (ie, Network Routers). No preference for either, but we may want to keep it consistent